README.md

```diff
@@ -15,4 +15,22 @@ make bundle-build
 podman push ${CONTAINER_REPO_OVERRIDE}/hyperfoil-operator-bundle:0.15.0
 # Creates CatalogSource and Subscription in the cluster
 operator-sdk run bundle ${CONTAINER_REPO_OVERRIDE}/hyperfoil-operator-bundle:0.15.0
-```
\ No newline at end of file
+```  
+
+## Deploy the operator and run Hyperfoil in minikube
+        
+First start by [installing minikube](https://minikube.sigs.k8s.io/docs/start/). Start the cluster with `m̀inikube start`. (*Suggestion:* use `minikube dashboard` to monitor the cluster)
+                       
+Build and deploy Hyperfoil operator (Go 1.19 is required) with `make build install deploy`
```

Comment:
```
[2/2] STEP 5/5: ENTRYPOINT ["/manager"]
[2/2] COMMIT quay.io/hyperfoil/hyperfoil-operator:0.21.0
--> 3649f5f3674a
Successfully tagged quay.io/hyperfoil/hyperfoil-operator:0.21.0
3649f5f3674af6cf23d15145555106b486db3d82582451e05504c5cdc69be1fd
docker push quay.io/hyperfoil/hyperfoil-operator:0.21.0
```

The docker push to quay.io is def wrong, so that needs fixing - push to local registry is ok

Reply:
Removed the deployment of the operator on it's own namespace, and instead added target to deploy resources in `config/samples`.